lib/components/molecules/Form/FieldToggle/FieldToggle.style.js

```diff
@@ -0,0 +1,70 @@
+import { css } from 'styled-components';
+
+const styles = css`
+  ${props => `
```

Comment:
I don't this this padding bottom should be handled from here as this styling should come from FORM container.